### PR TITLE
LPS-912966 / LPS-91298 / LPS-92700 Use Lexicon user icon

### PR DESCRIPTION
**Pull request by adolfopa**
Hey @julien,

In this PR I've changed the UserPortrait tag and Sharing app to use
the new Lexicon user icon.  Let me know if there's any problem with
this.

Thanks!

/cc @jbalsas @victorvalle

### CODE REVIEW

**Comment by julien** [comment]
Just started reviewing :)

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

File: modules/apps/sharing/sharing-web/src/main/resources/META-INF/resources/ManageCollaborators.soy [hunk 1]
```diff
@@ -102,11 +102,22 @@
...
 
 	<div class="autofit-col">
-		{call ClaySticker.render}
-			{param size: 'xl' /}
-			{param imageSrc: $collaborator.imageSrc /}
-			{param shape: 'circle' /}
-		{/call}
+		{if $collaborator.imageSrc}
+			{call ClaySticker.render}
+				{param size: 'xl' /}
```
Comment (ambrinchaudhary) [inline modules/apps/sharing/sharing-web/src/main/resources/META-INF/resources/ManageCollaborators.soy]: We should use the "lg" size here, same like when there is no portraitURL

**Pull request closed without merging**